### DISCUSSION TITLE
:has and :contains selector not working in 'find elements' of 'Elements' tab of Chrome Dev Tools

### DISCUSSION THREAD

**Discussion by rajndev** [jquery/jquery#5155] [Q&A] [answered]
### Description ###

When I type the following `jQuery` selector into the `find element` field of the `Elements` tab of Chrome dev tools (after typing Ctrl+F), it doesn't find/highlight the corresponding element in the DOM window. Doesn't Chrome support the `:has` or `:contains` selectors in its dev tools?

**Comment by mgol** [comment, marked as answer]
jQuery has multiple selector extensions so you can't expect jQuery selectors to work natively in the browser. In particular, the `:contains` pseudo-class is not supported natively.

**Reply by rajndev** [reply]
Ok. Thanks for the clarification.
